test/loader.js

```diff
@@ -0,0 +1,8 @@
+describe('Handlebars.helpers.loader', function() {
+
+    it('should return a div', function() {
```

Comment:
Unless I'm missing something, it would be Jasmine's Spies? i.e spyOn(DDG, "is3x").and.returnValue(true);